contracts/stake/src/state.rs

```diff
@@ -314,11 +337,13 @@ impl StakeState {
             );
         }
 
-        if stake.reward == 0 {
+        // Shift eligibility only if warnings are saturated
```

Comment:
In the documentation, we refer to this as "suspension", rather than "shifting".
It would be better to be consistent, but feel free to do as you prefer

Reply:
Done